requirements.txt

```
@@ -16,3 +16,4 @@ xarray-datatree==0.0.6
 psutil>=5.9.1
 geopy
 flox>=0.7.2,<1.0.0
+charset-normalizer==3.1.0
```

See my comment about this.
```suggestion
charset-normalizer<3.3
```